defaults/main.yml

```diff
@@ -12,62 +12,21 @@ owncloud_domain: [ 'owncloud.{{ ansible_domain }}' ]
 
 # --- ownCloud source and deployment ---
 
+owncloud_edition: community
+owncloud_release: 8.0
+owncloud_apt_repo_base: 'http://download.opensuse.org/repositories/isv:/ownCloud:'
+
 # User and group that will be used for ownCloud instance
-owncloud_user: 'owncloud'
-owncloud_group: 'owncloud'
-owncloud_home: '/srv/users/{{ owncloud_user }}'
+owncloud_user: 'www-data'
+owncloud_group: 'www-data'
+owncloud_home: '/var/www/owncloud'
```

Comment:
I would prefer ownCloud to be on its own system user and group for security reasons. `php5-fpm` can be used to create separate pool for that user. Also, `owncloud_home` should point to `'{{ ansible_local.root.app + "/owncloud" }}'`, which can then be mapped to different directories, by default it would be `/var/local/owncloud`.

I assume that these settings are hardcoded in the `.deb` ownCloud package (haven't seen the rest of the changes yet). I wonder if we could change that using debconf answers...